tensorflow/python/data/kernel_tests/interleave_test.py

```diff
@@ -124,28 +126,27 @@ def testPythonImplementation(self, input_values, cycle_length, block_length,
                                        block_length)):
       self.assertEqual(expected, produced)
 
-  @parameterized.named_parameters(
-      ("1", np.int64([4, 5, 6]), 1, 3, None),
-      ("2", np.int64([4, 5, 6]), 1, 3, 1),
-      ("3", np.int64([4, 5, 6]), 2, 1, None),
-      ("4", np.int64([4, 5, 6]), 2, 1, 1),
-      ("5", np.int64([4, 5, 6]), 2, 1, 2),
-      ("6", np.int64([4, 5, 6]), 2, 3, None),
-      ("7", np.int64([4, 5, 6]), 2, 3, 1),
-      ("8", np.int64([4, 5, 6]), 2, 3, 2),
-      ("9", np.int64([4, 5, 6]), 7, 2, None),
-      ("10", np.int64([4, 5, 6]), 7, 2, 1),
-      ("11", np.int64([4, 5, 6]), 7, 2, 3),
-      ("12", np.int64([4, 5, 6]), 7, 2, 5),
-      ("13", np.int64([4, 5, 6]), 7, 2, 7),
-      ("14", np.int64([4, 5, 6]), dataset_ops.AUTOTUNE, 3, None),
-      ("15", np.int64([4, 5, 6]), dataset_ops.AUTOTUNE, 3, 1),
-      ("16", np.int64([]), 2, 3, None),
-      ("17", np.int64([0, 0, 0]), 2, 3, None),
-      ("18", np.int64([4, 0, 6]), 2, 3, None),
-      ("19", np.int64([4, 0, 6]), 2, 3, 1),
-      ("20", np.int64([4, 0, 6]), 2, 3, 2),
-  )
+  @combinations.generate(
+      combinations.times(
+          test_base.default_test_combinations(),
+          combinations.combine(input_values=[np.int64([4, 5, 6])],
+                               cycle_length=1, block_length=3,
+                               num_parallel_calls=[None, 1]) +
+          combinations.combine(input_values=[np.int64([4, 5, 6])],
+                               cycle_length=2, block_length=[1, 3],
+                               num_parallel_calls=[None, 1, 2]) +
+          combinations.combine(input_values=[np.int64([4, 5, 6])],
+                               cycle_length=7, block_length=2,
+                               num_parallel_calls=[None, 1, 3, 5, 7]) +
+          combinations.combine(input_values=[np.int64([4, 5, 6, 7])],
```

Comment:
Change the original `input_values=[np.int64([4, 5, 6])]` to `input_values=[np.int64([4, 5, 6, 7])]`. Otherwise, there will be an error: 
```
absl.testing.parameterized.DuplicateTestNameError: Duplicate parameterized test name in InterleaveTest: generated test name 'testInterleaveDataset_test_mode_eager_tfapiversion_1_blocklength_3_cyclelength_1_inputvalues_456_numparallelcalls_None' (generated from 'testInterleaveDataset_test_mode_eager_tfapiversion_1_blocklength_3_cyclelength_1_inputvalues_456_numparallelcalls_None') already exists. Consider using named_parameters() to give your tests unique names and/or renaming the conflicting test method.
```
The cause is that when generating the test name, `dataset_ops.AUTOTUNE` will be converted to `1` instead of `-1`, which conflicts with the first combination (line 132 - 134)